twyla/service/message.py

```diff
@@ -56,19 +53,45 @@ class Meta(BaseModel):
     session_id: UUID = uuid4()
 
 
-class Content(BaseModel):
-    pass
+class IntegrationRequestContent(BaseModel):
```

Comment:
I'm a bit confused. Didn't we talk about not adding chatbot related fields? I was expecting a simple `EventPayload` model, and `content` and `context` as fields on the model.